packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -56,8 +58,29 @@ const l1Networks: { [chainId: number]: L1Network } = {
   }
 }
 
+export type BaseNetwork = L1Network & { isBase: true }
+
+const baseNetworks: { [chainId: number]: BaseNetwork } = {
+  [ChainId.Base]: {
+    chainId: ChainId.Base,
+    blockTime: 2,
+    isTestnet: false,
+    isBase: true
+  },
+  [ChainId.BaseSepolia]: {
+    chainId: ChainId.BaseSepolia,
+    blockTime: 2,
+    isTestnet: true,
+    isBase: true
+  }
+}
+
 export const getChains = () => {
-  const chains = [...Object.values(l1Networks), ...getArbitrumNetworks()]
+  const chains: (L1Network | ArbitrumNetwork | BaseNetwork)[] = [
+    ...Object.values(l1Networks),
+    baseNetworks[ChainId.BaseSepolia] as BaseNetwork,
```

Comment:
not including Base mainnet for now because no Orbit chains are on top of it yet

Reply:
We can include it but nothing will be added, so it's already implemented